src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/HtmlTag.java

```diff
@@ -23,7 +23,7 @@
  * Used to keep track of a tag and the text that follows it.
  *
  */
-class HtmlTag {
+/* default */ class HtmlTag {
```

Comment:
public.

The same if smb what to create it and use - it is ok. We will not support them , use such users should beware that class exists as is and will change then javadoc processing need this. We will support only code that is hosted in our repo, all other is not our problems (it is not contract to users to have any of this classes).